src/keychron/q3/ansi.json

```diff
@@ -0,0 +1,251 @@
+{
+  "name": "Keychron Q3",
+  "vendorId": "0x3434",
+  "productId": "0x0120",
+  "lighting": {
+    "extends": "qmk_rgblight",
+    "underglowEffects": [
+      ["00. None", 0],
+      ["01. SOLID_COLOR", 1],
+      ["02. BREATHING", 1],
+      ["03. BAND_SPIRAL_VAL", 1],
+      ["04. CYCLE_ALL", 1],
+      ["05. CYCLE_LEFT_RIGHT", 1],
+      ["06. CYCLE_UP_DOWN", 1],
+      ["07. RAINBOW_MOVING_CHEVRON", 1],
+      ["08. CYCLE_OUT_IN", 1],
+      ["09. CYCLE_OUT_IN_DUAL", 1],
+      ["10. CYCLE_PINWHEEL", 1],
+      ["11. CYCKE_SPIRAL", 1],
```

Comment:
There is a typo in all of the submitted files, the `CYCKE_SPIRAL` => `CYCLE_SPIRAL`

Reply:
Ty for pointing that out. Will correct the spelling when I can.

Reply:
@cTn-dev, pushed a commit to fix the spelling.